iis-wf/iis-wf-ingest-pmc/src/main/java/eu/dnetlib/iis/wf/ingest/pmc/metadata/ArticleMetaXmlHandler.java

```diff
@@ -0,0 +1,234 @@
+package eu.dnetlib.iis.wf.ingest.pmc.metadata;
+
+import static eu.dnetlib.iis.wf.ingest.pmc.metadata.PmcXmlConstants.*;
+import static eu.dnetlib.iis.wf.ingest.pmc.metadata.TagHierarchyUtils.*;
+
+import java.util.List;
+import java.util.Stack;
+
+import org.apache.commons.lang.StringUtils;
+import org.jdom.Element;
+import org.xml.sax.Attributes;
+import org.xml.sax.SAXException;
+import org.xml.sax.helpers.DefaultHandler;
+
+import com.google.common.collect.Lists;
+
+import eu.dnetlib.iis.common.importer.CermineAffiliation;
+import eu.dnetlib.iis.common.importer.CermineAffiliationBuilder;
+import eu.dnetlib.iis.ingest.pmc.metadata.schemas.Affiliation;
+import eu.dnetlib.iis.ingest.pmc.metadata.schemas.Author;
+import eu.dnetlib.iis.ingest.pmc.metadata.schemas.ExtractedDocumentMetadata;
+import eu.dnetlib.iis.ingest.pmc.metadata.schemas.Range;
+import pl.edu.icm.cermine.exception.AnalysisException;
+import pl.edu.icm.cermine.exception.TransformationException;
+import pl.edu.icm.cermine.metadata.affiliation.CRFAffiliationParser;
+
+/**
+ * Sax xml handler of &lt;article-meta&gt; tag in pmc xml
```

Comment:
...In JATS xml... I would say. We should avoid the expression pmc in a context like this, because pmc is just a db that uses the JATS format.